packages/nodes-base/nodes/Airtop/actions/file/upload.operation.ts

```diff
@@ -136,6 +148,7 @@ export async function execute(
 		index,
 		true,
 	) as boolean;
+	const elementDescription = this.getNodeParameter('elementDescription', index, '') as string;
```

Comment:
`const elementDescription = this.getNodeParameter('elementDescription', index, undefined) as string;` should the fallback be undefined?